Packs/EWS/Integrations/EWSO365/EWSO365_test.py

```diff
@@ -505,3 +510,66 @@ def test_parse_incident_from_item_with_attachments():
     )
     incident = parse_incident_from_item(message)
     assert incident['attachment']
+
+
+@pytest.mark.parametrize('params, expected_result', [
+    ({'_tenant_id': '_tenant_id', '_client_id': '_client_id', 'default_target_mailbox': 'default_target_mailbox'},
+     'Key / Application Secret must be provided.')
+])
+def test_key_params(mocker, params, expected_result):
+    """
+    Given:
+      - Configuration parameters
+    When:
+      - One of the required parameters are missed.
+    Then:
+      - Ensure the exception message as expected.
+    """
+
+    mocker.patch.object(demisto, 'params', return_value=params)
+    mocker.patch.object(demisto, 'error')
+    sub_main()
+
+    assert demisto.error.call_args[0][0] == "Exception: " + expected_result
+
+
+@pytest.mark.parametrize('params, expected_result', [
+    ({'_tenant_id': '_tenant_id', 'credentials': {'password': '1234'},
+      'default_target_mailbox': 'default_target_mailbox'}, 'ID / Application ID must be provided.')
+])
+def test_id_params(mocker, params, expected_result):
```

Comment:
let's merge back the 3 tests into one (as we understood the issue is not related to them being separate, correct me if I'm wrong)

Reply:
This was a different issue that was related to the logs, so the tests still have to be separated